valhalla/sif/dynamiccost.h

```diff
@@ -332,6 +333,15 @@ class DynamicCost {
   virtual bool IsExcluded(const baldr::GraphTile*& tile,
                           const baldr::NodeInfo* node);
 
+  // Adds a list of edges (GraphIds) to the user specified avoid list.
+  void AddUserAvoidEdges(const std::vector<baldr::GraphId>& avoid_edges);
```

Comment:
in case we ever have the notion of system wide avoids (e.g. daily construction closures). This method may go away when we pass info from loki to sif - but it allowed a way to test with valhalla_run_route.